Describe the new PivotGrid export API (#1164)
DevExpress/devextreme-documentation: api-reference/50 Common/Object Structures/ExportPivotGridProps/customizeCell.md

                        if(gridCell.rowType === 'D') {
                            excelCell.font = { color: { argb: 'FF0000FF' }, underline: true };
                            excelCell.alignment = { horizontal: 'left' };
                        }
                    }
                }).then(function() {
                    workbook.xlsx.writeBuffer().then(function(buffer) {
                        saveAs(new Blob([buffer], { type: "application/octet-stream" }), "Companies.xlsx");
                    });
                });
                e.cancel = true;
            }
        }
    }
    </script>

##### React

    <!-- tab: App.js -->
    import React from 'react';
    import 'devextreme/dist/css/dx.common.css';
    import 'devextreme/dist/css/dx.light.css';

    import PivotGrid, { Export } from 'devextreme-react/pivot-grid';
    import ExcelJS from 'exceljs';
    import saveAs from 'file-saver';
    import { exportPivotGrid } from 'devextreme/excel_exporter';

    export default function App() {
        return (
            <PivotGrid ...
                onExporting={onExporting}>
                <Export enabled={true} />
            </PivotGrid>
        );

>         function onExporting(e) {

Reviewer: It is better to define auxiliary functions outside the component function: https://stackoverflow.com/a/46138370

---
---
##### shortDescription
<!-- %shortDescription% -->

##### param(options): Object
<!-- %param(options)% -->

##### field(options.excelCell): Object
<!-- %field(options.excelCell)% -->

##### field(options.pivotCell): ExcelPivotGridCell
A **PivotGrid** cell.

---

> The following code illustrates how to customize <a href="https://github.com/exceljs/exceljs#fonts" target="_blank">font</a> and <a href="https://github.com/exceljs/exceljs#alignment" target="_blank">alignment</a> in cells whose [rowType](/Documentation/ApiReference/UI_Widgets/dxPivotGrid/Pivot_Grid_Cell/#rowType) is *"D"*:

Reviewer: ```suggestion
In the following code, the **customizeCell** function customizes <a href="https://github.com/exceljs/exceljs#fonts" target="_blank">font</a> and <a href="https://github.com/exceljs/exceljs#alignment" target="_blank">alignment</a> in cells that display regular summaries (whose [rowType](/Documentation/ApiReference/UI_Widgets/dxPivotGrid/Pivot_Grid_Cell/#rowType) is *"D"*):
```

---
##### jQuery

    <!--JavaScript-->
    $(function() {
        $("#pivotGridContainer").dxPivotGrid({
            // ...
            export: {
                enabled: true
            },
            onExporting(e) {

>                 var workbook = new ExcelJS.Workbook();
>                 var worksheet = workbook.addWorksheet('Companies');

Reviewer: ```suggestion
                const workbook = new ExcelJS.Workbook();
                const worksheet = workbook.addWorksheet('Companies');
```

                DevExpress.excelExporter.exportPivotGrid({
                    component: e.component,
                    worksheet: worksheet,
                    topLeftCell: { row: 2, column: 2 },
                    customizeCell: function(options) {

>                         var { gridCell, excelCell } = options;

Reviewer: It's strange that we use destructuring, but don't use constants although constants have wider support.
```suggestion
                        const { gridCell, excelCell } = options;
```

                    topLeftCell: { row: 2, column: 2 },
                    customizeCell: function(options) {
                        const { gridCell, excelCell } = options;
                        
                        if(gridCell.rowType === 'D') {
                            excelCell.font = { color: { argb: 'FF0000FF' }, underline: true };
                            excelCell.alignment = { horizontal: 'left' };
                        }
                    }
                }).then(function() {
                    workbook.xlsx.writeBuffer().then(function(buffer) {
                        saveAs(new Blob([buffer], { type: "application/octet-stream" }), "Companies.xlsx");
                    });
                });
                e.cancel = true;
            }
        }
    }
    </script>

##### React

    <!-- tab: App.js -->
    import React from 'react';
    import 'devextreme/dist/css/dx.common.css';
    import 'devextreme/dist/css/dx.light.css';

    import PivotGrid, { Export } from 'devextreme-react/pivot-grid';
    import ExcelJS from 'exceljs';
    import saveAs from 'file-saver';
    import { exportPivotGrid } from 'devextreme/excel_exporter';

>     class App extends React.Component {

Reviewer: We agreed to use hooks in new examples. It means that we should switch from class to functional components regardless of whether a particular example uses hooks or not.
